sites/oemmagazine/server/graphql/fragments/content-page.js

```diff
@@ -42,6 +42,18 @@ fragment ContentPageFragment on Content {
     credit
     isLogo
   }
+  images(input:{ pagination: { limit: 100 }, sort: { order: values } }) {
```

Comment:
Are all the content page fragments identical? If so, move them to the common folder with the rest of the common graphql fragments and update all the imports to use it

Reply:
@zarathustra323 I think i did this the right way :)